config/configuration.json

```diff
@@ -4,7 +4,7 @@
       "org.openstreetmap.atlas.checks.validation"
     ],
     "type": "org.openstreetmap.atlas.checks.base.BaseCheck",
-    "enabled.value.default": true
+    "enabled.value.default": false
```

Comment:
This needs to be reverted.

Reply:
addressed

Reply:
> I had a question and a couple of comments, otherwise this looks good. Also, are there any unit tests that need to be updated to capture what is being fixed in this PR?

I could technically add some unit tests for the process where edge proving escapable is being put through the "inconsistent" tag logic prior to continuing case 3 logic.

Reply:
at the moment I think the tests are fine but I will see if anymore are necessary